doc/classes/Node.xml

```diff
@@ -438,7 +438,30 @@
 			<return type="void">
 			</return>
 			<description>
-				Prints the scene to stdout. Used mainly for debugging purposes.
+				Prints the tree to stdout. Used mainly for debugging purposes. This version displays the path relative to the current node, and is good for copy/pasting into the get_node() function.
+				[code]
+TheGame
+TheGame/Menu
+TheGame/Menu/Label
+TheGame/Menu/Camera2D
+TheGame/SplashScreen
+TheGame/SplashScreen/Camera2D
+				[/code]
+			</description>
+		</method>
+		<method name="print_tree_pretty">
+			<return type="void">
+			</return>
+			<description>
+				Similar to print_tree, this prints the tree to stdout. This version displays a more graphical representation similar to what is displayed in the scene inspector. It is useful for inspecting larger trees.
```

Comment:
Use `[method print_tree]` to make an hyperlink to the method you're referring to.